docs/pages/accounts/personal.md

```diff
@@ -30,4 +30,4 @@ Common situations where access tokens are useful:
 
 ### Revoking Access Tokens
 
-In case a token was accidentally leaked, you can revoke them without changing your username and password. When you revoke the access token, you block all access to your account using this token. To do this, go to the Expo Dashboard Settings, at [https://expo.io/dashboard/ACCOUNT/settings/access-tokens](https://expo.io/dashboard/ACCOUNT/settings/access-tokens), and delete the token you want to revoke.
+In case a token was accidentally leaked, you can revoke them without changing your username and password. When you revoke the access token, you block all access to your account using this token. To do this, go to the [Expo Dashboard Settings access token page](https://expo.io/dashboard/[account]/settings/access-tokens), and delete the token you want to revoke.
```

Comment:
```suggestion
In case a token was accidentally leaked, you can revoke it without changing your username and password. When you revoke the access token, you block all access to your account using this token. To do this, go to the [Access Token page](https://expo.io/dashboard/[account]/settings/access-tokens) on your dashboard and delete the token you want to revoke.
```